### ISSUE TITLE
[a11y] 9.1.3.1d - Structured content

### ISSUE THREAD

**Issue by tbsbdr** [owncloud/web#5379]
- Context: The Web UI should be accessible according to the WCAG2.1 /BITV 2.0 guidelines. The number in the title relates to the test-steps: https://www.bitvtest.de/bitv_test/das_testverfahren_im_detail/pruefschritte.html
- "Status" relates to the evaluation scheme https://www.bitvtest.de/bitv_test/das_testverfahren_im_detail/verfahren.html#c1262 
- "a11y" is short for accessibility
- ----------------

Page 3 (Gelöschte Dateien)
The sentence "vor ein paar Sekunden" should be implemented with a structure element for paragraphs ( p ).

## Status: rather fulfilled
